src/components/Footer.js

```diff
@@ -1,8 +1,127 @@
+                    icon={faGithub}
+                  />
+                  <span class="ml-2"> Github </span>
+                </a>
+              </p>
+            </li>
+          </ul>
+        </section>
+
+        <section>
+          <h6 className="text-uppercase">Developers</h6>
+          <ul className="list-unstyled mb-1">
+            <li>
+              <p className="mb-0">
+                <a
+                  className="linkedin social text-white"
+                  href="https://www.linkedin.com/in/viktoriia-kosenko-030983168/"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <FontAwesomeIcon
+                    aria-hidden="true"
+                    aria-label="Linkedin icon"
+                    icon={faLinkedin}
+                    size="1x"
+                  />
+                  <span class="ml-2"> Vicky </span>
+                </a>
+              </p>
+            </li>
+            <li>
+              <p className="mb-0">
+                <a
+                  className="linkedin social text-white"
+                  href="https://www.linkedin.com/mwlite/in/ahmed8merehil"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <FontAwesomeIcon
+                    aria-hidden="true"
+                    aria-label="Linkedin icon"
+                    icon={faLinkedin}
+                    size="1x"
+                  />
+                  <span class="ml-2"> Ahmed </span>
+                </a>
+              </p>
+            </li>
+            <li>
+              <p className="mb-0">
+                <a
+                  className="linkedin social text-white"
+                  href="https://www.linkedin.com/in/abdulkadir-awel-23781a1a4/"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <FontAwesomeIcon
+                    aria-hidden="true"
+                    aria-label="Linkedin icon"
+                    icon={faLinkedin}
+                    size="1x"
+                  />
+                  <span class="ml-2"> Abdulkadir </span>
+                </a>
+              </p>
+            </li>
+            <li>
+              <p className="mb-0">
+                <a
+                  className="linkedin social text-white"
+                  href="#"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <FontAwesomeIcon
+                    aria-hidden="true"
+                    aria-label="Linkedin icon"
+                    icon={faLinkedin}
+                    size="1x"
+                  />
+                  <span class="ml-2"> Ali </span>
+                </a>
+              </p>
+            </li>
+          </ul>
+        </section>
+      </div>
+
+      <section className="bg-info text-center py-2">
+        <small>
```

Comment:
`small` should be applied to text within a `p`, so let's move it as a child of `p`